index.bs

```diff
@@ -348,13 +357,14 @@ To <dfn>maybe process an attribution source</dfn> given a <a spec="HTML">navigat
 To <dfn>process an attribution source</dfn> given an [=attribution source=] |source|:
 
 1. Let |cache| be the user agent's [=attribution source cache=].
-1. [=list/Remove=] all entries in |cache| where any of:
+1. [=list/Remove=] all entries in |cache| where all of the following are true:
     1. the entry's [=attribution source/attribution destination=] matches |source|'s [=attribution source/attribution destination=].
     1. the entry's [=attribution source/reporting endpoint=] matches |source|'s [=attribution source/reporting endpoint=].
```

Comment:
Sorry for missing this earlier: "matches" is a bad word to use in a spec unless it's linked to a definition, because it's unclear what equivalence relation it implies. You probably mean https://html.spec.whatwg.org/multipage/origin.html#same-origin here?

Reply:
Hmmm, for reportingendpoint same origin is correct.

However, attribution destination is actually a site (https://html.spec.whatwg.org/multipage/origin.html#site). The attribution source struct currently declares this as an origin which is wrong.

Would "the entries ... is equivalent to ..." work?

Reply:
No, "equivalent to" also has multiple definitions. For sites, I think you can use "are the same" or "are equal". (You'd use https://html.spec.whatwg.org/multipage/origin.html#same-site if it were an origin, but I don't see a defined term for sites, and their structure is simple enough that equality is right. (And hosts, the only complicated part, have https://url.spec.whatwg.org/#concept-host-equals that just uses "A is B".)

Reply:
Got it, thanks for the explanation!